workdir/openapi.yaml

```diff
@@ -18453,6 +18478,73 @@ paths:
     parameters:
     - $ref: '#/components/parameters/org_id'
     - $ref: '#/components/parameters/sso_id'
+  /api/v1/orgs/{org_id}/ssr/export_idtokens:
+    parameters:
+    - $ref: '#/components/parameters/org_id'
+    post:
+      description: While using Mist to onboard 128T devices, you can export the IDTokens
+        and import it into Conductor so it will only allow those devices to get securely
+        onboarded.
+      operationId: exportOrgSsrIdTokens
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/mac_addresses'
+        description: Request Body
+      responses:
+        '200':
+          $ref: '#/components/responses/SsrExportIdTokens'
+        '400':
+          $ref: '#/components/responses/HTTP400'
+        '401':
+          $ref: '#/components/responses/HTTP401'
+        '403':
+          $ref: '#/components/responses/HTTP403'
+        '404':
+          $ref: '#/components/responses/HTTP404'
+        '429':
+          $ref: '#/components/responses/HTTP429'
+      summary: exportOrgSsrIdTokens
+      tags:
+      - Orgs Devices - SSR
+  /api/v1/orgs/{org_id}/ssr/register_cmd:
+    get:
+      description: SSR devices can be managed/adopted by Mist.
+      operationId: getOrgSsrRegistrationCommands
+      parameters:
+      - description: duration for the token to stay valid,in seconds. Defaults to
+          1 year (31536000 seconds) if not specified.
+        in: query
+        name: ttl
+        schema:
+          example: 31536000
+          type: integer
+      - description: When specified, only specified assets are allowed for registration.
```

Comment:
When specified restricts registration to listed assets only. Prefer HTTP body over headers for this parameter, especially with long lists to avoid header size limits.